vittorio-corbo/Equinox
# Added pause menu, fixed start screen

Added a rudimentary pause menu and options menu and fixed the Start screen.

TO TEST:
Go into Scenes > GameScenes > Start, hit play. Start game. Hit P to pause, pause menu should come up. Hit P again or select Resume and you should be able to keep playing.

Options menu is available in both the main menu and when paused. Manipulating settings in either should affect the game's settings. The Quit button in the pause menu should bring you back to the Starting scene. Then you should be able to restart the game or quit (at least if this was built).

Note that the scene that hitting Start drops you in is the scene that I used to test the pause feature. Other scenes would need to be set up the same was as the scene named "pause", and the Main Menu would need the value of the scene to switch to changed.
